test/multiverse/suites/delayed_job/before_suite.rb

```diff
@@ -43,7 +44,7 @@ class CreateDelayedJobs
     @connection = $db_connection
   end
 
-  class CreatePelicans < ActiveRecord::Migration
+  class CreatePelicans < ActiveRecord::VERSION::STRING >= "5.0.0" ? ActiveRecord::Migration["#{ActiveRecord::VERSION::STRING[0]}.0"] : ActiveRecord::Migration
```

Comment:
Leaving like this for now, but there are lots of other references to the added ternary in other migrations. After this PR is merged, I'll create another that abstracts this out into a helper method and updates all references.